client/packages/host/src/components/Sync/Sync.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { PropsWithChildren, useState, useEffect } from 'react';
+
+import {
+  Grid,
+  LoadingButton,
+  RadioIcon,
+  Typography,
+  useFormatDateTime,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useHost } from '../../api/hooks';
+import { SyncProgress } from '../SyncProgress';
+
+const STATUS_POLLING_INTERVAL = 1000;
```

Comment:
> when you sync it updates in real time doesn't it?

Not sure what you mean (on server yeah it's up to date with sync steps), on f/e it updates based on polling interval (wanted to have polling interval from the start, in case sync is already ongoing while page is being opened (there is a test for page refresh in the tests to see this).

Ideally we would use graphql subscriptions for this, but haven't quite got the chance, added to cary over with low priority: https://github.com/openmsupply/open-msupply/issues/688

> We could probably dial it back a little

Sure how what should be the interval ? This would be the same for Initialisation ? (a bit of traffic).

Reply:
all good - I think leave it as 1s. I just noticed that my server was handling a lot of requests when I was on that screen. Fine for now.. and we can look at subscription / websockets etc later. maybe..